### PR TITLE
Prepare release 1.6.0

### PR DESCRIPTION
**Pull request by Sija**

### CODE REVIEW

**Comment by Sija** [comment]
@veelenga ping

File: shard.yml [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
-  # TODO: remove pre-compiled executable in future releases
   postinstall: shards build -Dpreview_mt
 
+# TODO: remove pre-compiled executable in future releases
 executables:
   - ameba
   - ameba.cr
 
-crystal: "~> 1.9.0"
+crystal: ~> 1.9
```
Comment (veelenga) [inline shard.yml:20]: since we drop the compatibility with crystal 1.9 below, shouldn't we bump it to 1.10?

Comment (Sija) [reply shard.yml:20]: We do? If yes, I missed that.

Comment (veelenga) [reply shard.yml:20]: We remove the code that ensures BC for crystal <= 1.9

https://github.com/crystal-ameba/ameba/pull/425/files#diff-3c764e731960f30a76c6f13b0387d5494855d0f52c8fa1a66df17637e3450680L79-L106

Comment (veelenga) [reply shard.yml:20]: Which is fine, however, I expected to bump the version here

```suggestion
crystal: ~> 1.10
```

Comment (Sija) [reply shard.yml]: Done 🧹

**Review by veelenga — APPROVED** [review]
LGTM :shipit:

**Pull request merged**